src/Components/NotesPage/NotesPage.jsx

```diff
@@ -86,13 +102,14 @@ const NotesPage = () => {
 							tagChangeHandler={tagChangeHandler}
 							priorityChangeHandler={priorityChangeHandler}
 							colorChangeHandler={colorChangeHandler}
+							updateNoteHandler={updateNoteHandler}
 						/>
 					</Modal>
 
 					{notes.length > 0 ? (
 						getFilterData(filterState).length > 0 ? (
```

Comment:
You can call the function only once and store the result in a variable.

Reply:
Done.